src/common/Widget.js

```diff
@@ -296,12 +296,14 @@
     };
 
     Widget.prototype.locateParentWidget = function (domNode) {
-        domNode = domNode || this._target.parentNode;
-        var widget = this.toWidget(domNode);
-        if (widget) {
-            return widget;
-        } else if (domNode.parentNode) {
-            return this.locateParentWidget(domNode.parentNode);
+        domNode = domNode || (this._target ? this._target.parentNode : null);
+        if (domNode) {
+            var widget = this.toWidget(domNode);
+            if (widget) {
+                return widget;
+            } else if (domNode.parentNode) {
```

Comment:
FYI - This "fix" is when this method is called prior to the widget ever being rendered on a page (in which case _target is null and it can have no parent widget).